ipatests/prci_definitions/nightly_ipa-4-9_previous.yaml

```diff
@@ -41,1759 +41,1639 @@ topologies:
+    priority: 100
     job:
       class: RunPytest
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_membermanager.py
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 1800
         topology: *master_1repl
 
-  testing-fedora/krbtpolicy:
-    requires: [testing-fedora/build]
+  fedora-previous-ipa-4-9/krbtpolicy:
+    requires: [fedora-previous-ipa-4-9/build]
     priority: 50
     job:
       class: RunPytest
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_krbtpolicy.py
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 4800
         topology: *master_1repl
 
-  testing-fedora/test_winsyncmigrate:
-    requires: [testing-fedora/build]
+  fedora-previous-ipa-4-9/test_winsyncmigrate:
+    requires: [fedora-previous-ipa-4-9/build]
     priority: 50
     job:
       class: RunADTests
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_winsyncmigrate.py
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 4800
         topology: *ad_master
 
-  testing-fedora/test_trust:
-    requires: [testing-fedora/build]
+  fedora-previous-ipa-4-9/test_trust:
+    requires: [fedora-previous-ipa-4-9/build]
     priority: 50
     job:
       class: RunADTests
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_trust.py
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 9000
         topology: *adroot_adchild_adtree_master_1client
 
-  testing-fedora/test_backup_and_restore_TestBackupAndRestoreTrust:
-    requires: [testing-fedora/build]
+  fedora-previous-ipa-4-9/test_backup_and_restore_TestBackupAndRestoreTrust:
+    requires: [fedora-previous-ipa-4-9/build]
     priority: 50
     job:
       class: RunPytest
       args:
-        build_url: '{testing-fedora/build_url}'
-        update_packages: True
+        build_url: '{fedora-previous-ipa-4-9/build_url}'
         test_suite: test_integration/test_backup_and_restore.py::TestBackupAndRestoreTrust
-        template: *testing-master-latest
+        template: *ci-ipa-4-9-previous
         timeout: 7200
         topology: *master_1repl
 
-  testing-fedora/test_adtrust_install:
-    requires: [testing-fedora/build]
-    priority: 50
+  fedora-previous-ipa-4-9/test_adtrust_install:
+    requires: [fedora-previous-ipa-4-9/build]
+    priority: 100
```

Comment:
usually tests are run with priority: 50, was it a mistake?

Reply:
I've copied the content from master branch's `nightly_latest.yaml`, no idea why there were set with `100` there. 
PR-CI will execute them before the rest of the tests, that would be the only "side effect".